app/controllers/case_workers/admin/management_information_controller.rb

```diff
@@ -34,6 +35,8 @@ def generate
 
   def validate_report_type
     return if Stats::StatsReport::TYPES.include?(params[:report_type])
+    return if params[:report_type] == 'reports_access_details'
```

Comment:
I had done this initially to avoid the (remote) possibility of this report type being used with the regenerate url. As it turns out, this didn't work as I expected so it might as well be in the `TYPE` array.

Reply:
I have added `report_access_details` to `Stats::StatsReport::TYPES` but this hasn't removed the requirement for the additional check as it should only work for the download url and not the generate url. It may be possible to do this better once #4377 has been merged into master and this PR gets rebased.

Reply:
A side effect of adding to `TYPES` is that it now appears visibly on the download page. It would be possible to fix this but I think it may benefit from the `Report` struct that is introduced in #4377 so I will wait for that before changing anything more.